web/src/engine/osk/src/visualKeyboard.ts

```diff
@@ -547,27 +560,56 @@ export default class VisualKeyboard extends EventEmitter<EventMap> implements Ke
           //
           // Merely constructing the instance is enough; it'll link into the sequence's events and
           // handle everything that remains for the backspace from here.
-          handler = new HeldRepeater(gestureSequence, () => this.modelKeyClick(gestureKey, coord));
+          handlers = [new HeldRepeater(gestureSequence, () => this.modelKeyClick(gestureKey, coord))];
         } else if(gestureStage.matchedId.indexOf('longpress') > -1) {
           // Matches:  'longpress', 'longpress-reset'.
           // Likewise.
-          handler = new SubkeyPopup(
+          handlers = [new SubkeyPopup(
             gestureSequence,
             configChanger,
             this,
             gestureSequence.stageReports[0].sources[0].baseItem,
             DEFAULT_GESTURE_PARAMS
-          );
+          )];
         } else if(baseItem.key.spec.multitap && (gestureStage.matchedId == 'initial-tap' || gestureStage.matchedId == 'multitap' || gestureStage.matchedId == 'modipress-start')) {
           // Likewise - mere construction is enough.
-          handler = new Multitap(gestureSequence, this, baseItem, keyResult.contextToken);
+          handlers = [new Multitap(gestureSequence, this, baseItem, keyResult.contextToken)];
+        }
+
+        if(gestureStage.matchedId.includes('modipress') && gestureStage.matchedId.includes('-start')) {
+          if(this.layerLocked) {
+            console.warn("Unexpected state:  modipress start attempt during an active modipress");
+          } else {
+            if(!handlers) {
+              handlers = [];
+            }
```

Comment:
```suggestion
            handlers = handlers ?? [];
```

Not terribly important though this one